src/main/java/de/muenchen/isi/domain/service/calculation/SobonursaechlicheWohneinheitenService.java

```diff
@@ -0,0 +1,141 @@
+     * Errechnet SoBoN-ursächliche Wohneinheiten je Förderart und Jahr.
+     * Ursprungsinfo ist die SoBoN-ursächliche Geschossfläche in der Abfragevariante
+     * Die Fördermixe aller Bauraten werden vor der Berechnung umgelegt, siehe {@link FoerdermixUmlageService}.
+     *
+     * Ist das Ergebnis aus dieser Rechnung über 1000 Wohneinheiten, werden die Wohneinheiten auf mehrere Bauraten (mehrere Jahre) aufgeteilt.
+     * Es wird immer bei 1000 WE ein Schnitt gemacht.
+     *
+     * Für die Summenbildung gibt es eine zusätzliche "Förderart", die die Summen aller Förderarten pro Jahr enthält.
+     * Außerdem gibt es zusätzliche Jahre pro Förderart, die über 10, 15 & 20 Jahre die Wohneinheiten aufsummieren.
+     *
+     *
+     * @param sobonGF SoBoN-ursächliche Geschossfläche in der Abfragevariante
+     * @param baurate Baurate (erstes Jahr). Werden mehrere Bauabschnitte angelegt, wird vom ersten Bauabschnitt,
+     *         der angelegt wurde, das erste Baugebiet, das angelegt wurde und dann davon die erste Baurate verwendet.
+     * @param sobonJahr Das SoBoN-Jahr, welches die städtebaulichen Orientierungswerte diktiert.
+     * @param gueltigAb Das Gültigkeitsdatum der Stammdaten, welche die Umlegung diktieren.
+     * @return Eine Liste von {@link WohneinheitenProFoerderartProJahrModel}, welche alle Wohneinheiten pro Förderart und Jahr darstellt.
+     */
+    public List<WohneinheitenProFoerderartProJahrModel> calculateSobonursaechlicheWohneinheiten(
+        final BigDecimal sobonGF,
+        final BaurateModel baurate,
+        final SobonOrientierungswertJahr sobonJahr,
+        final LocalDate gueltigAb
+    ) {
+        final var sobonsursachlicheWohneinheitenList = new ArrayList<WohneinheitenProFoerderartProJahrModel>();
+
+        // Umlegen von Förderarten
+        baurate.setFoerdermix(foerdermixUmlageService.legeFoerdermixUm(baurate.getFoerdermix(), gueltigAb));
+
+        // Berechnen der Wohneinheiten pro Förderart und Jahr
+        calculateWohneinheiten(baurate, 0, sobonGF, sobonJahr.getGueltigAb(), sobonsursachlicheWohneinheitenList);
+
+        // Summe der Wohneinheiten aller Förderarten
+        BigDecimal summeWE = BigDecimal.ZERO;
+        for (WohneinheitenProFoerderartProJahrModel sobonursaechlicheWE : sobonsursachlicheWohneinheitenList) {
+            summeWE = summeWE.add(sobonursaechlicheWE.getWohneinheiten());
+        }
+
+        // Ist das Ergebnis über 1000, werden 1000er-Blöcke/Jahre gebildet.
+        if (summeWE.intValue() > 1000) {
+            sobonsursachlicheWohneinheitenList.clear();
+
+            BigDecimal gfWohnen1000 = sobonGF.multiply(
+                new BigDecimal(1000).divide(summeWE, CalculationService.DIVISION_SCALE, RoundingMode.HALF_EVEN)
+            );
+            // Anzahl 1000er-Blöcke/Jahre = summeWE / 1000 = Ergebnis abrunden
+            Integer anzahl = summeWE
+                .divide(new BigDecimal(1000), CalculationService.DIVISION_SCALE, RoundingMode.HALF_DOWN)
```

Comment:
new BigDecimal(1000) als Konstante definieren, da mehrfache Verwendung in der Klasse und Instantiierung relativ Laufzeitaufwändig ist.